src/components/HookForm/index.jsx

```diff
@@ -344,10 +350,17 @@ export default class HookForm extends Component {
       hook: assocPath(['metadata', 'name'], e.target.value, this.state.hook),
     });
 
-  handleOwnerChange = e =>
+  handleOwnerChange = e =>{
     this.setState({
       hook: assocPath(['metadata', 'owner'], e.target.value, this.state.hook),
-    });
+      validation:{
+        owner: {
+          error: !e.currentTarget.validity.valid,
+          message: e.currentTarget.validationMessage,
+        },
+      },
+    })
+  };
 
   handleDescriptionChange = e =>
     this.setState({
```

Comment:
In the `validHook` method, you will want to include `!validation.owner.error`. See the diff.